.docker/images/nginx/tests/bootstrap.php

```diff
@@ -6,29 +6,48 @@
 
 include_once __DIR__ . '/vendor/autoload.php';
 
-function get_curl_headers($path = "/", $opts = NULL)
-{
-  $uri = getenv('LOCALDEV_URL') ?: 'http://nginx:8080';
```

Comment:
`LOCALDEV_URL` should never be used from inside of the containers as this variable is only intended to be used by the host OS when providing routing to containers (through pygmy, for example).